src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File read = new File(fromFileName);
+        File report = new File(toFileName);
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(read));
```

Comment:
consider using try-with-resources instead

also, please see checklist points about constants, exceptions and helper methods

Reply:
using try-with-resources everywhere, added constants, fixed exceptions